proof-market/application-developers/tutorials/signing-up.mdx

```diff
@@ -0,0 +1,39 @@
+import ApiSchema from '@theme/ApiSchema';
+
+# Signing up
+
+This tutorial showcases how authorization works in Proof Market and its API.
```

Comment:
This description and overview is lacking couple key things:

1. Explaining our sign schema, where we have. That we basically have 3 calls process, and the reason for it
2. Explain how to sign the message (it's 2 step) probbaly in multiple ways

Then we can just mention the steps with endpoints. But now it feels it's lacking overview